htlcswitch/interceptable_switch.go

```diff
@@ -171,10 +195,25 @@ func (s *InterceptableSwitch) Stop() error {
 	close(s.quit)
 	s.wg.Wait()
 
+	s.blockEpochStream.Cancel()
```

Comment:
Should this go before `Wait()`?

Reply:
I don't think it matters, because `run` has its own quit channel. I do it after `run` finishes, because the block subscription is also started before `run` starts.